### PR TITLE
Support for aggregate trades and raw trades in websocket interface.

### LINKED ISSUE #90: Should the `trades` stream delegate to the same endpoint rather than `aggTrades?
It seems that the `ws.trades` method delegates to the `aggTrades` endpoint:

https://github.com/HyperCubeProject/binance-api-node/blob/master/src/websocket.js#L165-L167

instead of using the `trades` endpoint as indicated in the readme:

https://github.com/HyperCubeProject/binance-api-node#trades-1

Is this a bug?

UPDATE: fixed the last link to point to WS rather than REST

### PR DESCRIPTION
**Pull request by hveiras**
closes #90

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1042,6 +1042,33 @@ client.ws.trades(['ETHBTC', 'BNBBTC'], trade => {
...
+  price: '0.04923600',
+  quantity: '3.43500000',
+  maker: false,
+  tradeId: 2148226
+}
+```
+
+</details>
+
+#### aggregate trades
```
Comment (balthazar) [inline README.md]: Can you use the name of the method, and add it to the list on top of the readme?

Comment (hveiras) [reply README.md]: Done

Comment (balthazar) [reply README.md]: And same thing on this line? Sorry to be annoying :P

**Comment by balthazar** [comment]
Perfect, thank you!

**Pull request closed without merging**